tests/pkcli_data/m1.in/x.py

```diff
@@ -0,0 +1,5 @@
+import pykern.pkcli
+import sys
+
+pykern.pkcli.message("this is a message")
```

Comment:
This is not the intended usage. It should only be called from a pkcli. Better to structure it that way so people don't use it as a example (tests as documentation).